packages/core/src/components/bal-date/bal-date.tsx

```diff
@@ -149,7 +149,12 @@ export class Date implements ComponentInterface, Loggable, BalAriaFormLinking {
   /**
    * Callback to determine which date in the datepicker should be selectable.
    */
-  @Prop({ attribute: 'allowed-dates' }) allowedDates: BalProps.BalDateCallback | undefined = undefined
+  @Prop() allowedDates: BalProps.BalDateCallback | undefined = undefined
+
+  /**
+   * If `true`, it returns the string `INVALID_VALUE` within the balChange event if the input provided is not valid.
+   */
+  @Prop() allowInvalidValue = false
```

Comment:
should we export a constant with the invalid string (as of now "INVALID_VALUE") and mention this in the docs so the value can be compared easier by users of the component?

Reply:
will do so